src/components/WorkersTable.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="workers-table">
+    <div class="workers-table__filters">
+      <ResultsCount label="Worker Pool" :count="filteredWorkers.length" class="worker-pools__results" />
+
+      <SearchInput v-model="searchValue" class="worker-pools__search" placeholder="Search worker pools" />
+    </div>
+
+    <p-table :data="filteredWorkers" :columns="columns">
+      <template #name="{ row }">
+        <span>{{ row.name }}</span>
+      </template>
+
+      <template #last-polled="{ row }">
+        <span>{{ formatDateTimeNumeric(row.lastHeartbeatTime) }}</span>
+      </template>
+
+      <template #action-heading>
+        <span />
+      </template>
+
+      <template #action="{ row }">
+        <div class="workers-table__actions">
+          <p-icon-button-menu>
+            <CopyOverflowMenuItem label="Copy ID" :item="row.id" />
+          </p-icon-button-menu>
+        </div>
+      </template>
+
+      <template #empty-state>
+        <PEmptyResults>
+          <template #message>
+            No workers
+          </template>
+          <template v-if="hasFilters" #actions>
+            <p-button size="sm" secondary @click="clear">
+              Clear Filters
+            </p-button>
+          </template>
+        </PEmptyResults>
+      </template>
+    </p-table>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed, ref } from 'vue'
+  import { ResultsCount, SearchInput, CopyOverflowMenuItem } from '@/components'
+  import { useWorkspaceApi } from '@/compositions'
+  import { formatDateTimeNumeric } from '@/utilities/dates'
+
+  const props = defineProps<{
+    workerPoolName: string,
+  }>()
+
+  const api = useWorkspaceApi()
+  const subscriptionOptions = {
+    interval: 30000,
+  }
+
+  const workerPoolNameRef = computed(() => props.workerPoolName)
```

Comment:
@stackoverfloweth has started using `toRefs` rather than computed props and I kinda like that. You don't have to change this but just raising awareness. 

```typescript
const { workerPoolName } = toRefs(props)
```

Reply:
Is there a benefite of one over the other?

Reply:
No not really. I'd say toRefs is a little more concise especially when there are multiple props. But they both accomplish the same thing. I don't think we have to pick one or the other.

Reply:
+1